app/models/shelter.rb

```diff
@@ -31,4 +32,12 @@ def alphabetical_pets
   def shelter_pets_filtered_by_age(age_filter)
     adoptable_pets.where("age >= ?", age_filter)
   end
+
+  def self.reverse_alphabetical_order
+    find_by_sql("SELECT * FROM shelters ORDER BY name desc")
+  end
+
+  def self.pending_apps
```

Comment:
I assume these are the methods changed back to passing tests? They look a little different than the original but I ran the tests and they all work, so nice job refactoring.